schema_based_on_diagram.sql

```diff
@@ -38,9 +38,17 @@ create table invoice_items(
     foreign key (treatment_id) REFERENCES treatments(id)
 )
 
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/221995963-b6872713-2fce-4e7f-9cf1-66d6cc9a44c8.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/221996317-ff79c9ed-db48-48b1-ac50-3a9af5b837e0.png)
## 3.
![image](https://user-images.githubusercontent.com/93039359/221996358-8bca801b-0e65-43eb-9c75-aa7415881b95.png)
## 4.
![image](https://user-images.githubusercontent.com/93039359/221996400-ed706f91-2eeb-48db-8b04-37e8dad1a2e0.png)
## 5.
![image](https://user-images.githubusercontent.com/93039359/221996447-66403e37-b4e9-4de5-a755-1ddee0a0d7e3.png)
## 6.
![image](https://user-images.githubusercontent.com/93039359/221996495-eda13fc0-abd3-445b-bbf9-ada330f43d1a.png)
## 7.
![image](https://user-images.githubusercontent.com/93039359/221996545-9e382b4a-26fb-44e2-8693-a9a079cac24a.png)
**[Optional]**

- [ ] Kindly remove all these section descriptions from your readme file to make your readme file clean and descriptive besides these section descriptions are just instructions for the owner of the repository :thinking: so this doesn't make any sense if your repository viewers will see all these instructions :thinking: